web/src/beta/components/StorytellingPageSectionItem/index.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from "@emotion/styled";
+import { FC } from "react";
+
+import Icon from "../Icon";
+import Text from "../Text";
+
+type Props = {
+  icon: string;
+  title: string;
+  index: number;
+  active?: boolean;
+  onAction?: () => void;
+  onClick?: () => void;
+};
+
+const StorytellingPageSectionItem: FC<Props> = ({
+  icon,
+  title,
+  index,
+  active,
+  onAction,
+  onClick,
+}) => {
+  return (
+    <HorizontalBox>
+      <VerticalBox>
+        <Text size={"m"} color="#c7c5c5">
+          {index}
+        </Text>
+        <Icon icon={icon} />
+      </VerticalBox>
+      <TitleArea active={active}>
+        <Text onClick={onClick} size={"s"} color="#ffffff">
+          {title}
+        </Text>
+        <Icon icon={"actionbutton"} onClick={onAction} />
```

Comment:
It is better to wrap it with `button` or something like a clickable HTML tag.
Also needs `cursor: pointer` CSS. ( if wrap it with `button`, it might be unnecessary )